src/sql/workbench/browser/modelComponents/inputbox.component.ts

```diff
@@ -269,6 +274,14 @@ export default class InputBoxComponent extends ComponentBase<azdata.InputBoxProp
 		this.setPropertyFromUI<string>((props, value) => props.placeHolder = value, newValue);
 	}
 
+	public get title(): string {
+		return this.getPropertyOrDefault<string>((props) => props.title, '');
+	}
+
+	public set title(newValue: string) {
```

Comment:
Why do we need a public setter for this? I do not see any clients or use- cases of this.

Reply:
This is the standard for all ModelView components - it's exposed so that main thread side things can update properties directly. We currently don't use these often but there's no reason not to include it.